frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js

```diff
@@ -1,5 +1,186 @@
 
-export default function ProofOfCompletedServicd_Web(props) {
-  return <div>Proof of Completed Service Web</div>;
+export default function ProofOfCompletedService_Web({ data }) {
+  return (
+    <FormThatPrints>
+      <Section name="Personal Information">
+        {/* <TextInput dataKey="person.firstName" label={__("First Name")} />
+        <TextInput dataKey="person.middleName" label={_("Middle Name")} />
+        <TextInput dataKey="person.lastName" label={_("Last Name")} />
+        <TextInput dataKey="person.addressStreet" label={_("Street Address")} />
+        <TextInput dataKey="person.addressCity" label={_("City")} />
+        <TextInput dataKey="person.addressState" label={_("State")} />
+        <TextInput dataKey="person.addressZip" label={_("Zip Code")} />
+        <TextInput dataKey="person.phone" label={_("Phone")} />
+        <TextInput dataKey="person.email" label={_("Email")} /> */}
+        <TextInput dataKey="person.firstName" label="First Name" />
+        <TextInput dataKey="person.middleName" label="Middle Name" />
+        <TextInput dataKey="person.lastName" label="Last Name" />
+        <TextInput dataKey="person.addressStreet" label="Street Address" />
+        <TextInput dataKey="person.addressCity" label="City" />
+        <TextInput dataKey="person.addressState" label="State" />
+        <TextInput dataKey="person.addressZip" label="Zip Code" />
+        <TextInput dataKey="person.phone" label="Phone" />
+        <TextInput dataKey="person.email" label="Email" />
+        <small className="web-form-input">
+          Check your email. You will receive information and documents at this
+          email address.
+        </small>
+      </Section>
+
+      <Section name="Case Information">
+        <Radio
+          dataKey="case.courtType"
+          label="Court Type"
+          options={courtTypeOptions}
+        />
+
+        {data.case.courtType === "District" && (
+          <GroupSelect
+            dataKey="case.courtAddress"
+            label="District Courts"
+            groupOptions={DistrictCourtList}
+          />
+        )}
+        {data.case.courtType === "Justice" && (
+          <GroupSelect
+            dataKey="case.courtAddress"
+            label="Justice Courts"
+            groupOptions={JusticeCourtList}
+          />
+        )}
+        <TextInput dataKey="case.caseNumber" label="Case Number" />
+        <TextInput dataKey="case.judgeName" label="Judge Full Name" />
+        <TextInput
+          dataKey="case.commissionerName"
+          label="Commissioner (Domestic Cases)"
+        />
+      </Section>
+
+      <Section name="1. Documents Served">
+        <small className="web-form-input">
+          The following documents were served by the method described below
+        </small>
+        <small className="web-form-input">(Choose all that apply:)</small>
+        <Checkbox
+          dataKey="proofOfCompletedService_case.summons"
```

Comment:
In data keys, the part before the period represents the Firebase collection for the data key. We currently only have two firebase collections - `case` and `person`. We should not introduce new ones without careful consideration. See [this related comment](https://github.com/JustUtahCoders/utahexpungements.org/pull/109#discussion_r341833909) which explains this more.

```suggestion
          dataKey="case.proofOfServiceSummons"
```

Reply:
Switched from proofOfCompletedService_case to case